src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -249,27 +265,43 @@ class PrisonerIepLevelReviewService(
 
   private suspend fun getIepLevelForReviewType(prisonerInfo: PrisonerAtLocation, reviewType: ReviewType): String {
     val iepLevelsForPrison = iepLevelService.getIepLevelsForPrison(prisonerInfo.agencyId, useClientCredentials = true)
+    val defaultLevel = iepLevelsForPrison.find(IepLevel::default) ?: iepLevelsForPrison.first() // if no default use the lowest sequence
```

Comment:
Is it "normal" for prisons to not have a default IEP level?

If they should always have a default IEP level I think it's worth keeping [that exception that is now removed](https://github.com/ministryofjustice/hmpps-incentives-api/pull/213/files#diff-8bf11d470f9e689ef0ce56f887d92fb6c5c77d56adf2a909d15d4a8d00096a64L253) as it highlights a data problem with an individual prison.

Reply:
But Nomis doesn't do that - so will be out of sync

Reply:
Yes